src/Package/Fetch.zig

```diff
@@ -1184,6 +1184,7 @@ fn unpackTarball(f: *Fetch, out_dir: fs.Dir, reader: anytype) RunError!UnpackRes
                 .unable_to_create_file => |i| res.unableToCreateFile(stripRoot(i.file_name, res.root_dir), i.code),
                 .unable_to_create_sym_link => |i| res.unableToCreateSymLink(stripRoot(i.file_name, res.root_dir), i.link_name, i.code),
                 .unsupported_file_type => |i| res.unsupportedFileType(stripRoot(i.file_name, res.root_dir), @intFromEnum(i.file_type)),
+                .components_outside_stripped_prefix => {}, // impossible with strip_components = 0
```

Comment:
If this cannot be reached (from how I understand the comment), it could/should be explicit `unreachable`, right?

Reply:
Right!

Was thinking about that but I somehow decide to go this way. But you are right we have keyword for marking unreachable code paths. Fixing it.
Thanks.